content/en/docs/ops/configuration/telemetry/envoy-stats/index.md

```diff
@@ -63,10 +63,11 @@ spec:
     defaultConfig:
       proxyStatsMatcher:
         inclusionRegexps:
-          - ".*circuit_breakers.*"
-        inclusionPrefixes:
-          - "upstream_rq_retry"
-          - "upstream_cx"
+          - ".*outlier_detection.*"
+          - ".*upstream_rq_retry.*"
+          - ".*upstream_cx_.*"
+        inclusionSuffixes:
+          - "upstream_rq_timeout"
```

Comment:
Looking for input from @istio/wg-policies-and-telemetry-maintainers

Reply:
https://github.com/istio/api/pull/2592 has not been merged yet. I suggest holding off updating the doc updates until the API changes and the implementation have been merged.

Reply:
The doc updates will happen when the istio/api PR merges as there is a nightly job which will scrape them into preliminary.istio.io.

Reply:
https://github.com/istio/api/pull/2592 has been merged.

Reply:
please wait https://github.com/istio/istio.io/pull/12383, then rebase from master, thanks~

Reply:
@zirain already rebase.